types/hydrogen.js.flow

```diff
@@ -1,3 +1,5 @@
+/* @flow */
```

Comment:
A long time ago I had this issue where I could not get hydrogen to load or test to run. I spent all sorts of time looking into it and reinstalled atom even. It turned out that I was using `// @flow` instead of `/* @flow */` or other way around and atom only supported one of the two. So we should be consistent with the one we use in all files

Mainly commenting so you don't have the same issue sometime (maybe atom fixed this)

Reply:
Thanks for telling me your experience, it must be so annoying 😅 
Maybe the recent version of Atom already fixed this, but anyway I would fix them.